README.md

```diff
@@ -1,2 +0,0 @@
-# bosh-softlayer-baremetal-server-release
```

Comment:
Changed this to a link to `src/github.com/cloudfoundry-community/vps/README.md`, please find it here.